### DISCUSSION TITLE
[RELEASE] Compact

### DISCUSSION THREAD

**Discussion by Paindeluxe** [nzp-team/nzportable#175] [Map Releases]
This is a Map that got Ported over to the newest NZP Version before I ported it, it was unable to be played on the newest Version.

I'll let [Dreize](https://www.moddb.com/members/dreize) the map creator explain his map.

> Here is another map I have made. It is called Compact, the reasoning behind that is because you are in a very "compact" space, you'll understand what I mean when you play it. This map was designed specifically for easy use with the psp. Its a camp around the box style map and it shouldn't present much lag as other maps do on the psp. My goal was to make a challenging map that worked well with the psp. Please note that this map can also be played just as well on the pc.
> 
> The difficulty level out of 10:
> 8/10
> 
> This map includes:
> -Juggernog
> -Speed Cola
> -PHD Flopper
> -Double Tap
> -Pack A Punch
> -Dogs

A Member of the NZP Discord called Ellis243454 requested me to port this over so eh credits to him for making me port this.

But of course the main Credits go to [Dreize](https://www.moddb.com/members/dreize) link to the original [Page](https://www.moddb.com/groups/nazi-zombies-portable-maps/addons/nzp-custom-map-compact)

### [DOWNLOAD](https://github.com/nzp-team/nzportable/files/10609724/Compact.zip)

**Comment by MotoLegacy** [comment]
Hey there! This map isn't eligible for the "Meets Install Standard" label, you can check out the requirements and why it's useful on the post for it: https://github.com/nzp-team/nzportable/discussions/159

**Reply by Paindeluxe** [reply]
Fixed! Sorry 'bout that.
